Bio/SeqIO/InsdcIO.py

```diff
@@ -602,9 +602,14 @@ def _write_the_first_line(self, record):
         if len(locus) > 16:
             if len(locus) + 1 + len(str(len(record))) > 28:
                 # Locus name and record length to long to squeeze in.
-                raise ValueError("Locus identifier %r is too long" % locus)
-            else:
-                warnings.warn("Stealing space from length field to allow long name in LOCUS line", BiopythonWarning)
```

Comment:
I had to read this part of the diff several times.

You're removing the old warning about stealing spaces (where the later fields would not change position), since this is now clearly allowed by the NCBI and ought not to break other parsers trying to read our output.

You're downgrading the old ValueError into a new warning (because this NCBI change is new, a warning makes sense as not every parser out there will cope).

I'm happy 👍